pandas/tests/indexes/test_numeric.py

```diff
@@ -886,3 +886,83 @@ def test_ufunc_coercions(self):
         tm.assertIsInstance(result, Float64Index)
         exp = Float64Index([0.5, 1., 1.5, 2., 2.5], name='x')
         tm.assert_index_equal(result, exp)
+
+
+class TestIndexCoercion(tm.TestCase):
+
+    _multiprocess_can_split_ = True
```

Comment:
and these too (as these are mixed Series/index things), IOW not particular to the index tests, but more so indexing